stylesheet/styles.css

```diff
@@ -0,0 +1,114 @@
+* {
+  box-sizing: border-box;
+}
+body {
+  margin-top: 0px;
+  margin-bottom: 0px;
+  margin-left: 0px;
+  margin-right: 0px;
+}
+header {
+  background-color: black;
+  color: white;
+  padding-bottom: 1rem;
+}
+img {
+  width: 100%;
+}
+p {
+  padding: 1rem;
+}
+h1 {
+  text-align: center;
+}
+footer {
+  clear: both;
+  text-align: center;
+  background-color: #093c60;
+  color: white;
+}
+.home-intro {
+  width: 25%;
+  float: left;
+}
+.intro-content {
+  width: 75%;
+  float: right;
+}
+.style-nav ul{
+  list-style-type: none;
+  padding: 2rem;
+}
+ .style-nav ul li a {
+   text-decoration: none;
+   color: grey;
+   text-align: center;
+   display: block;
+   text-transform: uppercase;
+   padding: 1rem;
+   font-size: 1rem;
+   float: left;
+   margin-bottom: 2rem;
+ }
+.style-nav ul li a:hover {
+  color: red;
+}
+.button {
+  border: 2px white solid;
+  padding: 8px 30px;
+  background-color: blue;
+  color: white;
+  text-decoration: none;
+
+}
+.button:hover {
+  background-color: grey;
+}
+.adopt-more-button {
+  border: 2px white solid;
+  padding: 8px 30px;
+  background-color: blue;
+  color: white;
+  text-decoration: none;
+}
+.adopt-more-button:hover {
+  background-color: grey;
+  border: 2px black solid;
+  color: black;
+}
+.adopt-more {
+  margin-top: 2rem;
+  margin-bottom: 2rem;
+  padding: 8px 30px;
+  color: white;
+  text-decoration: none;
+  text-align: center;
+}
+.col {
+  margin-top: 1rem;
+  text-align: center;
+  margin-bottom: 1rem;
+
+}
+.row {
```

Comment:
Your row/col system doesn't seem to work. The tiles are full width on my laptop. You'll want to use flexbox or grid to lay them out. A single wrapper around them might be helpful. Especially as we might not want to have 4 tiles wide on laptop, 3 on tablet and 1 on mobile. Flexbox/grid will be helpful